### PR TITLE
Duplicate fixes

### PR DESCRIPTION
**Pull request by mliu49**
### Motivation or Problem
We were not properly marking duplicates for reaction libraries in the edge and irreversible reactions.

### Description of Changes
Add duplicate checking when loading reaction libraries and update `markDuplicateReactions` to account for irreversible reactions.

### Testing
Check that the edge mechanism from the new sulfur test in RMG-tests can be loaded in Chemkin.

It would probably also be good to check RMG-tests to see if this has any noticeable effect on performance.

### CODE REVIEW

**Comment by alongd** [comment]
@mliu49, can you check why this fails?
The tests are too verbos, the raw log has many `ImportError`s and `AttributeError`s

**Comment by mliu49** [comment]
It seems like new numpy packages were released, which somehow broke a lot of things. I restarted the build to see if it was a temporary incompatibility issue. If not, I'll look more into what the conflict is.

**Comment by mliu49** [comment]
1. Anaconda uploaded new numpy packages (1.14)
2. Anaconda now splits the numpy package into numpy and numpy-base (no idea why and what the difference is)
3. We are currently restricted to numpy 1.11 (#1417), but could install numpy-base 1.14
4. The current quantities is incompatible with whatever modules are in numpy-base 1.14 (similar to #1335)
5. Updating quantities seems to fix the issue
6. However, we're still installing numpy 1.11 and numpy-base 1.14, which may or may not cause other issues

File: rmgpy/rmg/model.py [hunk 1]
```diff
@@ -1591,6 +1591,14 @@ def addReactionLibraryToEdge(self, reactionLibrary):
                 # Don't add to the edge library reactions that were already processed
                 self.addReactionToEdge(rxn)
 
+        if self.saveEdgeSpecies:
+            from rmgpy.chemkin import markDuplicateReaction
+            newEdgeReactions = self.edge.reactions[numOldEdgeReactions:]
+            checkedReactions = self.core.reactions + self.edge.reactions[:numOldEdgeReactions]
+            for rxn in newEdgeReactions:
+                markDuplicateReaction(rxn, checkedReactions)
```
Comment (alongd) [inline rmgpy/rmg/model.py]: Could you remind me why we don't need to `checkedReactions()` when adding a seed library to core?

File: rmgpy/chemkinTest.py [hunk 2]
```diff
@@ -381,6 +381,51 @@ def test_process_duplicate_reactions(self):
         self.assertTrue(isinstance(rtest.kinetics, MultiArrhenius))
         self.assertTrue(all(isinstance(k, Arrhenius) for k in rtest.kinetics.arrhenius))
 
+    def test_mark_duplicate_reactions(self):
```
Comment (alongd) [inline rmgpy/chemkinTest.py]: Excellent test implementation

File: rmgpy/rmg/model.py (hunk 1 above)
Comment (mliu49) [reply rmgpy/rmg/model.py]: In `addSeedMechanismToCore`, we call `self.markChemkinDuplicate` which calls `Chemkin.markDuplicateReactions` to process all reactions and mark duplicates. Here, we assume that the existing reactions have already been marked properly, so we just check the new reactions against the previously checked ones.

**Pull request merged**